deploy/manifests/config-map.yaml

```diff
@@ -13,7 +13,7 @@ data:
         "nodeQuery": |
           sum by (<<.GroupBy>>) (
             irate(
-                container_cpu_usage_seconds_total{<<.LabelMatchers>>,id='/'}[4m]
+                node_cpu_usage_seconds_total{<<.LabelMatchers>>}[4m]
```

Comment:
To do that we would need to install node-exporter. Although this is what is done in most setup these days, I would rather limit the amount of dependencies of this repo to just Prometheus and Kubernetes